### DISCUSSION TITLE
You don't have access to github Copilot.Join ......  [why?]

### DISCUSSION THREAD

**Discussion by JoengSeoiFung** [community/community#7655] [Copilot Conversations]
You don't have access to github Copilot.Join ......  [why?]

**Comment by davecheney** [comment]
We’re adding people as we have capacity, thanks for your patience and understanding.

**Comment by bagustris** [comment]
I have access to copilot from the beginning, but today I lost it. The vscode warning message said, "You don't have access to GitHub copilot". But I checked on the copilot page that I am already on the list. Why? I checked on four machines with different OS (windows and Linux) and all my copilots are gone. Why?

**Reply by davecheney** [reply]
I've confirmed you have an active invitation to use copilot. Can you please provide more details, screenshots would be ideal.

**Reply by bagustris** [reply]
Here is a screenshot from my Ubuntu 20.04 machine.  The log says below.
~~~
[INFO] [auth] [2022-01-27T01:22:22.444Z] Invalid copilot token: missing token: 401 
[ERROR] [default] [2022-01-27T01:22:22.445Z] GitHub Copilot could not connect to server. Extension activation failed: "User not authorized"
~~~
![image](https://user-images.githubusercontent.com/3158361/151274507-1e8c1942-1b69-4f0f-ae7d-b2c9ced444bb.png)

**Reply by davecheney** [reply]
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

**Reply by bagustris** [reply]
Thanks! I tried this several times since last night with no success. Now it works after I signed out, closed VScode, opened VScode again, and sign in. Weird, it happens in all of my four machines.
